### PR TITLE
Doc fixes

### LINKED ISSUE #120: Issue endpoint has incorrect description for "held_by_pokemon"
The documentation for v2 says this regarding the `held_by_pokemon` field on the item endpoint:
image

While in reality the structure is a bit different:
image

### LINKED ISSUE #121: baby_trigger_for is incorrectly labeled as being a list (or returned incorrectly as a single object)
image

I am working on the c# wrapper over at https://gitlab.com/PoroCYon/PokeApi.NET and set up a test that calls every all 746 items.
`baby_trigger_for` does not seem to be an array in any of them.

### LINKED ISSUE #131: Moves: mislabeled property
The docs have the following for the `move` endpoint. Marked is the issue
image

Per [this page](http://pokeapi.co/docsv2/#super-contest-effects) the second entry should be `super_contest_effect` which is am APIResource(SuperContestEffect).

Proof:
image

What skills would I need to just make a PR to the docs?

### LINKED ISSUE #132: Moves: contest_combos is not a list
Docs:
image

Actual return value:
image

### PR DESCRIPTION
**Pull request by M-Zuber**
- Fixes #131 
- Fixes #132 
- Fixes #121 
- Fixes #120

### CODE REVIEW

File: templates/pages/docsv2.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
 					<li><a href="#berry-flavors">Berry Flavors</a></li>
 					<li><a href="#characteristics">Characteristics</a></li>
 					<li><a href="#contest-types">Contest Types</a></li>
-					<li><a href="#contest-effects">Contest Effects</a></li>
+					<li><a href="#contest-effects">Contest Effects</a></li>ba
```
Comment (phalt) [inline templates/pages/docsv2.md]: `ba` at the end of this line needed?

**Comment by phalt** [comment]
:+1:

File: templates/pages/docsv2.md (hunk 1 above)
Comment (M-Zuber) [inline templates/pages/docsv2.md]: Nope. I have no idea how it got there

**Comment by phalt** [comment]
@M-Zuber wow that was fast

**Comment by M-Zuber** [comment]
You caught me at a good time ¯\_(ツ)_/¯

**Comment by M-Zuber** [comment]
I added a fix for the last issue of mine. Not that sure about it, but would be happy to fix up any issues you have with my writing.

**Comment by M-Zuber** [comment]
@phalt how does it look now?

**Pull request merged**